connector/dialect.tdd

```diff
@@ -725,6 +358,21 @@
       <argument type='real' />
       <argument type='real' />
     </function>
+    <function group='operator' name='!=' return-type='bool'>
+      <formula>(%1 &lt;&gt; %2)</formula>
```

Comment:
I notice parentheses around a lot of the formulas -- are they strictly necessary? Is this valid:
```
<formular>%1 &lt;&gt; %2</formula>
```
?

Reply:
I tested it and the functions I ran succeeded without parentheses.  I removed them.